report.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'board'
+require_relative 'burndown'
+require_relative 'helpers'
+
+module Spregen
+  class Report
+
+    include Spregen::Helpers
+
+    attr_accessor :report
+
+    def initialize(configuration = {})
+      Spregen::Config.configure_trello_client
+      @board = Spregen::Board.new(configuration[:board_name])
```

Comment:
I don't like "Board" as the class name. "Board" is only used for a specific client (trello).
Perhaps we can use something more generic, like Source. What do you think?